library/PlugLoad-windows.cpp

```diff
@@ -41,6 +41,8 @@ distribution.
  */
 namespace DFHack
 {
+    //! \todo What windows HMODULE matches RTLD_DEFAULT?
+    DFLibrary* GLOBAL_NAMES = 0;
```

Comment:
From what I've read, it seems to be `GetModuleHandle(NULL)`.

Reply:
Sure, but this should only be necessary on Linux (for now, anyway)

Reply:
I don't think there's any downside to using `GetModuleHandle(NULL)` right now, since it'll just add a reference to the current process that will go away when the current process exits anyway. It'll make it easier if we need it in the future.

Reply:
How is windows dfhack loading done?
I'm wondering if GetModuleHandle(NULL) is safe in static variable initialization